blazor/gantt-chart/virtualization.md

```diff
@@ -13,159 +13,118 @@ Gantt Chart allows you to load a large amount of data without performance degrad
 
 ## Row virtualization
 
-Row virtualization allows you to render rows only in the content viewport on load time. It is an alternative way of paging in which the rows will be loaded while scrolling vertically. To set up the row virtualization, you need to define `EnableVirtualization` as true and content height by [Height](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.SfGantt-1.html#Syncfusion_Blazor_Gantt_SfGantt_1_Height) property.
+The `EnableRowVirtualization` property allows you to render only the rows that are visible in the content viewport at load time. Rows are loaded while scrolling vertically, which optimizing memory usage by rendering only the rows that are visible, resulting in faster rendering and scrolling and efficiently handling large datasets in your Gantt chart without sacrificing performance or user experience. To set up row virtualization using this API, simply set [EnableRowVirtualization](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.SfGantt-1.html#Syncfusion_Blazor_Gantt_SfGantt_1_EnableRowVirtualization) to true.
 
-The number of records displayed in the Gantt Chart is determined implicitly by the height of the content area.
+The number of records displayed in the Gantt chart is determined implicitly by the height of the content area.
```

Comment:
if make c in chart chart in small case. dont we need to change g in gantt also?

Reply:
No, we don't need to change "g" in "Gantt" to lowercase. The capitalization of "Gantt" is specific to the term and its associated charting technique. Even if "chart" is written in lowercase, "Gantt" should still be capitalized to maintain consistency and ensure proper recognition of the Gantt chart as a distinct entity.